python3Packages.ipython: 8.4.0 -> 8.8.0

aaronjheng opened the pull request:
###### Description of changes

<!--
For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

aaronjheng committed:
python3Packages.ipython: 8.4.0 -> 8.10.0

nixos-discourse commented:
This pull request has been mentioned on **NixOS Discourse**. There might be relevant details there:

https://discourse.nixos.org/t/prs-ready-for-review/3032/1765

mweinelt commented on pkgs/development/python-modules/ipython/default.nix:
```diff
@@ -21,33 +20,22 @@
 , traitlets
 
-# Test dependencies
+  # Test dependencies
```
Please remove the reindentation of the comments.

aaronjheng replied:
They are `nixpkg-fmt`-ed.

azuwis committed:
helmfile: 0.145.2 -> 0.150.0

superherointj committed:
python311Packages.aiocache: 0.11.1 -> 0.12.0

Release: https://github.com/aio-libs/aiocache/releases/tag/v0.12.0

Diff: https://github.com/aio-libs/aiocache/compare/0.11.1...v0.12.0

SharzyL committed:
circt: 1.29.0 -> 1.30.0

Release notes: https://github.com/llvm/circt/releases/tag/firtool-1.30.0

fabaff committed:
python311Packages.aiocache: update meta

- disable on unsupported Python releases

fabaff committed:
python311Packages.aiocache: add optional-dependencies

fabaff committed:
python311Packages.aiocache: fix module name

StillerHarpo committed:
kodi.packages.invidious: 0.1.0+matrix.1 -> unstable-2022-11-28

e1mo committed:
nixos/dokuwiki: remove last reference to aclUse

This was overlooked in #209299, but will throw a trace when building a
system even when the setting was not set.

superherointj committed:
python311Packages.pyflunearyou: fix build

Commit:
synology-drive-client: 3.2.0 -> 3.2.1

figsoda committed:
felix-fm: 2.2.4 -> 2.2.5

Diff: https://github.com/kyoheiu/felix/compare/v2.2.4...v2.2.5

Changelog: https://github.com/kyoheiu/felix/blob/v2.2.5/CHANGELOG.md

r-ryantm committed:
python310Packages.fastbencode: 0.1 -> 0.2

fabaff committed:
qovery-cli: 0.48.6 -> 0.49.0

Diff: https://github.com/Qovery/qovery-cli/compare/v0.48.6...v0.49.0

Changelog: https://github.com/Qovery/qovery-cli/releases/tag/v0.49.0

aanderse committed:
Merge pull request #215792 from StillerHarpo/kodi/invidious

kodi.packages.invidious: 0.1.0+matrix.1 -> unstable-2022-11-28

NickCao committed:
Merge pull request #215885 from figsoda/felix

felix-fm: 2.2.4 -> 2.2.5

NickCao committed:
Merge pull request #215762 from SharzyL/circt-1.30.0

circt: 1.29.0 -> 1.30.0

NickCao committed:
Merge pull request #215850 from MoritzBoehme/synology-drive-client

synology-drive-client: 3.2.0 -> 3.2.1

NickCao committed:
Merge pull request #215894 from fabaff/qovery-cli-bump

qovery-cli: 0.48.6 -> 0.49.0

NickCao committed:
Merge pull request #215739 from superherointj/pkg-python311Packages.aiocache-0.12.0

python311Packages.aiocache: 0.11.1 -> 0.12.0

NickCao committed:
Merge pull request #215452 from azuwis/helmfile

helmfile: 0.145.2 -> 0.150.0

marsam committed:
Merge pull request #215891 from r-ryantm/auto-update/python310Packages.fastbencode

python310Packages.fastbencode: 0.1 -> 0.2

NickCao committed:
Merge pull request #215795 from e1mo/fix-dokuwiki-acluse

nixos/dokuwiki: remove last reference to aclUse

ncfavier commented:
Why did you close, @aaronjheng?

aaronjheng commented:
This PR is kinda messed up. Closing in favor of #215904.
